save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt

```diff
@@ -0,0 +1,471 @@
+/**
```

Comment:
The logic of split was different. We need to make a common community framework constructor:
- Try to create organization View in common 
- We will be able to provide different enums as confiuration on the implementation side (where you will be able to select whcih tabs are needed)

Reply:
Fixed